packages/devtools/src/commands/switchToFrame.js

```diff
@@ -32,7 +44,7 @@ export default async function switchToFrame ({ id }) {
          * `page` has `frames` method while `frame` has `childFrames` method
          */
         let getFrames = page.frames || page.childFrames
-        const childFrames = await getFrames()
+        const childFrames = await getFrames.apply(page)
```

Comment:
Why not do getFrames(page)?

Reply:
The method doesn't take arguments. @takeya0x86 is correct here we call the method without its scope.